eden/scm/tests/test-git-clone-sets-publicheads.t

```diff
@@ -0,0 +1,29 @@
+#chg-compatible
```

Comment:
nit: could you try `#debugruntest-compatible`? we are recommending using it for new tests.